### PR TITLE
Release 12.6.3 - update hashicorp/http

### PR DESCRIPTION
**Pull request by briskt**
### Fixed
- Replace deprecated `body` attribute with `response_body` in http data source.
- Removed extraneous comma in PASSWORD_RULE_minScore variable name.
- Removed duplicated AWS_REGION variable in 031-email-service task definition.

### CODE REVIEW

File: terraform/031-email-service/task-definition-api.json [hunk 1]
```diff
@@ -25,10 +25,6 @@
         "name": "APP_ID",
         "value": "${appconfig_app_id}"
       },
-      {
```
Comment (mtompset) [inline terraform/031-email-service/task-definition-api.json:28]: Why remove this?

File: terraform/041-id-broker-search-lambda/main.tf [hunk 2]
```diff
@@ -29,7 +29,7 @@ resource "aws_iam_role_policy_attachment" "AWSLambdaVPCAccessExecutionRole" {
 resource "aws_lambda_function" "search" {
   s3_bucket        = var.function_bucket_name
   s3_key           = var.function_zip_name
-  source_code_hash = data.http.function-checksum.body
+  source_code_hash = data.http.function-checksum.response_body
```
Comment (mtompset) [inline terraform/041-id-broker-search-lambda/main.tf:32]: I'm assuming newer version of lambda renamed things?

File: terraform/050-pw-manager/task-definition-api.json [hunk 3]
```diff
@@ -170,7 +170,7 @@
         "value": "${password_rule_minlength}"
       },
       {
-        "name": "PASSWORD_RULE_minScore,",
+        "name": "PASSWORD_RULE_minScore",
```
Comment (mtompset) [inline terraform/050-pw-manager/task-definition-api.json:173]: Nice catch. Did it even work before?

File: terraform/031-email-service/task-definition-api.json (hunk 1 above)
Comment (briskt) [reply terraform/031-email-service/task-definition-api.json:28]: It's duplicated. You'll see the same entry on (old) lines 52-55.

File: terraform/041-id-broker-search-lambda/main.tf (hunk 2 above)
Comment (briskt) [reply terraform/041-id-broker-search-lambda/main.tf:32]: Yes, but not lambda, it's the http data source. Both names are supported in versions 2 and 3 of the provider, but `body` is deprecated.

File: terraform/050-pw-manager/task-definition-api.json (hunk 3 above)
Comment (briskt) [reply terraform/050-pw-manager/task-definition-api.json:173]: No. From what I recall, we haven't tried to change this from the default value.

**Pull request merged**